File: KMTracker/KMTracker/Car.cs
using System.Collections.Generic;

namespace KMTracker
{
	public class Car
	{
		public string Name
		{
			get;
		}

		public string NumberPlate{
			get;
		}

		public double MileAge
		{
			get;
		}

		public List<Rit> Ritten
		{
			get;
		}

		public Car(string name, string numberPlate, double mileAge)
		{
			Ritten = new List<Rit>();

			Name = name;
			MileAge = mileAge;
			NumberPlate = numberPlate;
		}

		public override string ToString()
		{
			return string.Format("Car: {0}, \nNumberPlate: {1}, \nMileage: {2}, \nRitten: {3}", Name, NumberPlate, MileAge, Ritten.Count);
		}

	}
}

File: Server/api/models/Car.js
/**
 * Car.js
 *
 * @description :: TODO: You might write a short summary of how this model works and what it represents here.
 * @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
 */

module.exports = {

  autoUpdatedAt: false,
  autoCreatedAt: false,

  attributes: {

    name: {
      type: 'string',
      required: true
    },

    numberPlate: {
      type: 'string',
      required: true,
      primaryKey: true
    },

    mileAge: {
      type: 'string'
    },

    //Add reference to Ritten
    ritten: {
      collection: 'rit',
      via: 'car'
    }
  }
};

